management-microservice/src/main/java/com/nearsoft/fgaribay/mgmt/model/ProductEntity.java

```diff
@@ -0,0 +1,67 @@
+package com.nearsoft.fgaribay.mgmt.model;
+
+import javax.persistence.*;
+
+@Entity
+@Table(name = "products")
+@NamedStoredProcedureQueries({
+  @NamedStoredProcedureQuery(
+      name = "getAllProducts",
+      procedureName = "retrieve_products",
+      resultClasses = ProductEntity.class),
+  @NamedStoredProcedureQuery(
+      name = "createProduct",
+      procedureName = "create_product",
+      parameters = {
+        @StoredProcedureParameter(name = "p_id", mode = ParameterMode.IN, type = Long.class),
```

Comment:
`mode` is always `ParameterMode.IN` by default in this annotation.

Nitpick: Be consistent in the way you format your annotations XD